.travis.yml

```diff
@@ -6,15 +6,15 @@ os:
 language: go
 
 go:
-- "1.13.x"
```

Comment:
@stealthybox Would it make sense to keep both golang versions for testing?

Reply:
This is already merged, but I don't believe it would make sense to matrix test go versions.
The releases are only published with one specific version of go.

Thanks for reading!